src/web/mjs/connectors/KissComic.mjs

```diff
@@ -55,7 +55,11 @@ export default class KissComic extends Connector {
         const script = `
             new Promise((resolve, reject) => {
                 setTimeout(() => {
-                    resolve(lstImages);
+                    try {
+                        resolve(Array.from(document.querySelectorAll('#divImage img')).map(img => img.src));
```

Comment:
Consider to get full size images for blogspot links by e.g., replacing the `=s1600` end of the URL pathname with `=s0`

Reply:
Done, but I honestly think that was quite unnecessarily as the URLs are already set to `=s0` since the `quality` parameter has been set to `hq`. You can check it out on a page like [this one](https://readcomiconline.li/Comic/Batman-Three-Jokers/Issue-1?id=175017&readType=1&quality=hq)

Reply:
I wonder if 1 second wait is enough for big chapters or full comics issues.

Reply:
Yes, it is enough. Just to be clear: this script was working fine until a month or so ago. I’ve used it for lots of comics, and I used this updated version for even more. This PR is just fixing the issue which made it fail. Also, the one second wait is not to download the whole chapter. All that is needed is just the HTML of the page to be loaded. Honestly I don’t know what the one second is for exactly, I can imagine it might still work if removed. But still: I don’t feel like changing the code which seemed to work without a problem